### PR TITLE
ADD base for increasing and decreasing font size with keys

### PR DESCRIPTION
**Pull request by Loreno10**
PR not finished!

I followed the steps from https://github.com/felixse/FluentTerminal/issues/720

THe thing I struggle with is "Extend XtermTerminalView with a call to this method". I do not know how to achieve this. In what way should I extend it?

Also, I'm not sure about "add some new defaults to DefaultValueProvider". `DefaultValueProvider` already contains method to get `TerminalOptions` - this object already contains `FontSize`. Is there a need to add anything to `DefaultValueProvider` then?

### CODE REVIEW

File: FluentTerminal.App.ViewModels/TerminalViewModel.cs [hunk 1]
```diff
@@ -157,6 +160,10 @@ public void Restore(string data)
             PasteCommand = new AsyncCommand(Paste);
             CopyLinkCommand = new AsyncCommand(() => CopyTextAsync(HoveredUri), () => !string.IsNullOrWhiteSpace(HoveredUri));
             ShowSearchPanelCommand = new RelayCommand(() => ShowSearchPanel = true, () => !ShowSearchPanel);
+            IncreaseFontSizeCommand = new AsyncCommand(IncreaseFontSize, CanExecuteCommand);
+            DecreaseFontSizeCommand = new AsyncCommand(DecreaseFontSize, () => CanExecuteCommand() && FontSize > 1);
+
+            FontSize = _terminalOptions.FontSize;
```
Comment (Loreno10) [inline FluentTerminal.App.ViewModels/TerminalViewModel.cs:166]: Not sure about this one

File: FluentTerminal.App.ViewModels/TerminalViewModel.cs [hunk 2]
```diff
@@ -264,6 +271,11 @@ public TabThemeViewModel BackgroundTabTheme
 
         public ICommand ShowSearchPanelCommand { get; private set; }
 
+        public ICommand IncreaseFontSizeCommand { get; private set; }
+
+        public ICommand DecreaseFontSizeCommand { get; private set; }
```
Comment (Loreno10) [inline FluentTerminal.App.ViewModels/TerminalViewModel.cs:276]: Where do we actually bind these commands to something?

Comment (felixse) [reply FluentTerminal.App.ViewModels/TerminalViewModel.cs:276]: We don't need commands in this case since the actions will be handled by xterm.js itself

**Comment by felixse** [comment]
> PR not finished!
> 
> I followed the steps from #720
> 
> THe thing I struggle with is "Extend XtermTerminalView with a call to this method". I do not know how to achieve this. In what way should I extend it?

Something like `ChangeThemeAsync`. A method to call the method in the webview
 
> Also, I'm not sure about "add some new defaults to DefaultValueProvider". `DefaultValueProvider` already contains method to get `TerminalOptions` - this object already contains `FontSize`. Is there a need to add anything to `DefaultValueProvider` then?

You already added the new keybindings there. That's what this was about 😃

**Pull request merged**